### If You Use ConnecTF In Your Own Work, Please Cite Us:

- Brooks, M.D., C.-L. Juang, M. Katari, R. Tuteja, and G.M. Coruzzi, ConnecTF: A repository and tool for analyzing and integrating transcription factor regulation data. (In preparation)

### Data Sources

- Root TARGET
    - Brooks, M.D., J. Cirrone, A.V. Pasquino, J.M. Alvarez, J. Swift, S. Mittal, Juang C.-L., K. Varala, R.A. Gutiérrez, G. Krouk, D. Shasha, and G.M. Coruzzi, Network Walking charts transcriptional dynamics of nitrogen signaling by integrating validated and predicted genome-wide interactions. Nature Communications, 2019. 10(1): p. 1569.
    - Alvarez, J.M., A.L. Schinke, M.D. Brooks, A. Pasquino, L. Leonelli, K. Varala, A. Safi, G. Krouk, A. Krapp, and G.M. Coruzzi, Transient genome-wide interactions of the master transcription factor NLP7 initiate a rapid nitrogen response cascade. Nature Communications, 2020. (in press)
- Shoot TARGET/N-responsive genesets
    - Varala, K., A. Marshall-Colón, J. Cirrone, M.D. Brooks, A.V. Pasquino, S. Léran, S. Mittal, T.M. Rock, M.B. Edwards, G.J. Kim, S. Ruffel, W.R. McCombie, D. Shasha, and G.M. Coruzzi, Temporal transcriptional logic of dynamic regulatory networks underlying nitrogen signaling and use in plants. Proceedings of the National Academy of Sciences, 2018. 115(25): p. 6494-6499.
- DAP-seq
    - O'Malley, R.C., S.C. Huang, L. Song, M.G. Lewsey, A. Bartlett, J.R. Nery, M. Galli, A. Gallavotti, and J.R. Ecker, Cistrome and Epicistrome Features Shape the Regulatory DNA Landscape. Cell, 2016. 166(6): p. 1598.
- ChIP-seq
    - Song, L., S.C. Huang, A. Wise, R. Castanon, J.R. Nery, H. Chen, M. Watanabe, J. Thomas, Z. Bar-Joseph, and J.R. Ecker, A transcription factor hierarchy defines an environmental stress response network. Science, 2016. 354(6312): p. aag1550.
    - Birkenbihl, R. P., B. Kracher, M. Roccaro and I. E. Somssich. Induced Genome-Wide Binding of Three Arabidopsis WRKY Transcription Factors during Early MAMP-Triggered Immunity. The Plant Cell, 2017. 29(1): p. 20-38.
- TF-TF protein-protein interactions
    - Trigg, S.A., R.M. Garza, A. MacWilliams, J.R. Nery, A. Bartlett, R. Castanon, A. Goubil, J. Feeney, R. O'Malley, S.C. Huang, Z.Z. Zhang, M. Galli, and J.R. Ecker, CrY2H-seq: a massively multiplexed assay for deep-coverage interactome mapping. Nature Methods, 2017. 14(8): p. 819-825.
    - Yazaki, J., M. Galli, A. Y. Kim, K. Nito, F. Aleman, K. N. Chang, A.-R. Carvunis, R. Quan, H. Nguyen, L. Song, J. M. Alvarez, S.C. Huang, H. Chen, N. Ramachandran, S. Altmann, R. A. Gutiérrez, D. E. Hill, J. I. Schroeder, J. Chory, J. LaBaer, M. Vidal, P. Braun and J. R. Ecker, Mapping transcription factor interactome networks using HaloTag protein arrays. Proceedings of the National Academy of Sciences, 2016. 113(29): p. E4238-E4247. 
- Open Chromatin (DHS/ATAC)
    - Sullivan, A.M., Andrej A. Arsovski, J. Lempe, K.L. Bubb, M.T. Weirauch, P.J. Sabo, R. Sandstrom, R.E. Thurman, S. Neph, A.P. Reynolds, A.B. Stergachis, B. Vernot, A.K. Johnson, E. Haugen, S.T. Sullivan, A. Thompson, F.V. Neri, III, M. Weaver, M. Diegel, S. Mnaimneh, A. Yang, T.R. Hughes, J.L. Nemhauser, C. Queitsch and J.A. Stamatoyannopoulos, Mapping and Dynamics of Regulatory DNA and Transcription Factor Networks in A. thaliana. Cell Reports, 2014. 8(6): p. 2015-2030.
    - Lu, Z., A. P. Marand, W. A. Ricci, C. L. Ethridge, X. Zhang and R. J. Schmitz, The prevalence, evolution and chromatin signatures of plant regulatory elements. Nature Plants, 2019. 5(12): p. 1250-1259.


### Tools

- Poultney, C.S., R.A. Gutiérrez, M.S. Katari, M.L. Gifford, W.B. Paley, G.M. Coruzzi, and D.E. Shasha, Sungear: interactive visualization and functional analysis of genomic datasets. Bioinformatics, 2006. 23(2): p. 259-261.
